### PR TITLE
New example: Scroll snap

### PR DESCRIPTION
**Pull request by estelle**
similar to https://codepen.io/estelle/pen/JjmOpvm

### CODE REVIEW

File: modules/scroll_snap.html [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+                max="2"
+                value="1"
+                list="places"
+                id="inline"
+              />
+              inline position</label
+            >
+          </p>
+          <p>
+            <label><input type="checkbox" id="stop" /> Stop on all boxes</label>
```
Comment (bsmth) [inline modules/scroll_snap.html:170]: Is this clearer?
```suggestion
            <label><input type="checkbox" id="stop" /> disable skipping boxes</label>
```

Comment (bsmth) [reply modules/scroll_snap.html:170]: or "scrolling past boxes"?

File: modules/scroll_snap.html [hunk 2]
```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>Scroll snap</title>
+    <style>
+      li {
```
Comment (bsmth) [inline modules/scroll_snap.html:8]: editor warning here about empty ruleset in case we want to add something in there, defaults, maybe?

**Review by bsmth — APPROVED** [review]
Two minor comments, but leaving a +1 - thanks @estelle 👍🏻

File: modules/scroll_snap.html (hunk 2 above)
Comment (estelle) [reply modules/scroll_snap.html:8]: It's a JS hook.

File: modules/scroll_snap.html (hunk 1 above)
Comment (estelle) [inline modules/scroll_snap.html:170]:
```suggestion
            <label><input type="checkbox" id="stop" /> Prevent scrolling past boxes</label>
```

**Pull request merged**